app/src/main/java/lt/vilnius/tvarkau/fragments/MultipleProblemsMapFragment.kt

```diff
@@ -54,7 +58,7 @@ class MultipleProblemsMapFragment : BaseMapFragment(),
             Timber.e(throwable)
         }
 
-        legacyApiService.getProblems(request)
+        subscription = legacyApiService.getProblems(request)
```

Comment:
But your way breaks nice flow of Rx.

For example if there were multiple possible subscriptions one could use
```
  .....
  .subscribe()
  .apply{ subscriptions += this }
```